### DISCUSSION TITLE
install livewire 3

### DISCUSSION THREAD

**Discussion by aliqasemzadeh** [livewire/livewire#6189] [1. Help]
Hi 
when I run
`composer require livewire/livewire "^3.0@beta"`
I get
````
 Problem 1
    - Root composer.json requires livewire/livewire 3.0@beta, found livewire/livewire[dev-throw-error-when-testing-non-livewire-class, ..., dev-bad-hotfix-test, v0.0.1, ..., v0.7.4, v1.0.0, ..., 1.x-dev, v2.0.0, ..., 2.x-dev, v3.0.0-beta.1, ..., v3.0.0-beta.7] but it does not match the constraint.

````

**Comment by thatal** [comment]
You can also use `composer require livewire/livewire "^3.0.*@beta"` it will install the latest beta version.

**Comment by PerryvanderMeer** [comment]
Please take a look at https://github.com/livewire/livewire/pull/6076#issuecomment-1656336235 to see which command you can use.
